src/middlewared/middlewared/plugins/pool.py

```diff
@@ -3187,7 +3187,7 @@ async def do_create(self, app, data):
             data['acltype'] = 'NFSV4'
             data['aclmode'] = 'RESTRICTED'
 
-        if data['type'] == 'FILESYSTEM':
+        if data['type'] == 'FILESYSTEM' and data.get('acltype', 'INHERIT') != 'INHERIT':
```

Comment:
I may be missing something, but I can only see the code working for when `acltype` is omitted. If it's set to `INHERIT`, won't the check fail and _*not*_ go into the conditional ? 🤔

Reply:
Right, in this case we're skipping check if acltype is omitted (we're inheriting from parent implicitly or not changing it). Behavior is as follows:
* SKIP if ZVOL
* SKIP if acltype omitted from payload
* SKIP if acltype is set to INHERIT

Reply:
👍🏼